src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,34 +1,76 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
-
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
-
-      <span className="icon is-right" style={{ pointerEvents: 'all' }}>
-        {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-        <button
-          data-cy="clearSearchButton"
-          type="button"
-          className="delete"
+import React from 'react';
+
+import { FilterBy } from '../../types/FilterBy';
+
+const filterByOptions = Object.values(FilterBy);
+
+type Props = {
+  query: string;
+  onQueryChange: (newQuery: string) => void;
+  filterBy: string;
+  onFilterChange: (newFilterBy: FilterBy) => void;
+};
+
+export const TodoFilter: React.FC<Props> = ({
+  query,
+  filterBy,
+  onQueryChange,
+  onFilterChange,
```

Comment:
Consider renaming the 'onFilterChange' prop to 'onStatusFilterChange' to better match its purpose and usage in the component 🤔